client/src/secret/types.rs

```diff
@@ -156,7 +159,7 @@ pub struct InputSigningDataDto {
     pub output: OutputDto,
     /// The output metadata
     #[serde(rename = "outputMetadata")]
-    pub output_metadata: OutputMetadata,
+    pub output_metadata: OutputMetadataDto,
```

Comment:
Are we 100% sure this doesn't cause any issue with stored data ?

Reply:
Checked in the wallet, there we only use `InputSigningData` and not the DTO and also that we only create during the input selection, but don't store it